### PR TITLE
Fix a typo

### PR DESCRIPTION
**Pull request by patrocinio**
>

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/nodelocaldns.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ This page provides an overview of NodeLocal DNSCache feature in Kubernetes.
...
 
-NodeLocal DNSCache improves Cluster DNS performance by running a dns caching agent on cluster nodes as a Daemonset. In today's architecture, pods in ClusterFirst DNS mode reach out to a kube-dns serviceIP for DNS queries. This is translated to a kube-dns/CoreDNS endpoint via iptables rules added by kube-proxy. With this new architecture, pods will reach out to the dns caching agent running on the same node, thereby avoiding iptables DNAT rules and connection tracking. The local caching agent will query kube-dns service for cache misses of cluster hostnames(cluster.local suffix by default).
+NodeLocal DNSCache improves Cluster DNS performance by running a dns caching agent on cluster nodes as a DaemonSet. In today's architecture, pods in ClusterFirst DNS mode reach out to a kube-dns serviceIP for DNS queries. This is translated to a kube-dns/CoreDNS endpoint via iptables rules added by kube-proxy. With this new architecture, pods will reach out to the dns caching agent running on the same node, thereby avoiding iptables DNAT rules and connection tracking. The local caching agent will query kube-dns service for cache misses of cluster hostnames(cluster.local suffix by default).
```
Comment (ryanmcginnis) [inline content/en/docs/tasks/administer-cluster/nodelocaldns.md]: could you change "pods" to "Pods" throughout?

Comment (patrocinio) [reply content/en/docs/tasks/administer-cluster/nodelocaldns.md]: Yes, will do

Comment (patrocinio) [reply content/en/docs/tasks/administer-cluster/nodelocaldns.md]: Yes, will do. Thanks

**Pull request closed without merging**